message.go

```diff
@@ -224,6 +224,7 @@ func (msg *Authenticate) MessageType() MessageType {
 // [GOODBYE, Details|dict, Reason|uri]
 type Goodbye struct {
 	Details map[string]interface{}
+	Request ID
```

Comment:
This isn't up to [the spec](https://tools.ietf.org/html/draft-oberstet-hybi-tavendo-wamp-02#section-6.4).

It looks like you just wanted it for error reporting in client.go?